txmongo/collection.py

```diff
@@ -1238,49 +1238,67 @@ def on_proto(proto):
 
         return self._database.connection.getprotocol().addCallback(on_proto)
 
-    def _delete(self, filter, multi, _deadline):
+    @defer.inlineCallbacks
+    def _delete(
+        self, filter: dict, let: Optional[dict], multi: bool, _deadline: Optional[float]
```

Comment:
Please add tests for newly added `let` argument. Or at least add it to the plan in `notes.md`.

Reply:
test added in TestDeleteMany